bentoml/cli/bento.py

```diff
@@ -74,35 +75,57 @@ def _print_bento_table(bentos, wide=False):
     _echo(table_display)
 
 
-def _print_bentos_info(bentos, output_type):
+def _print_bentos_info(bentos, output_type, print_location=False):
     if output_type == 'table':
         _print_bento_table(bentos)
     elif output_type == 'wide':
         _print_bento_table(bentos, wide=True)
     else:
         for bento in bentos:
+            if print_location:
```

Comment:
Only printing a list of path seems not that useful to the user, currently the user can already use `bentoml list -o wide` or `bentoml get IrisClassifier -o wide` to view the list of saved bundle alongside with the saved location.  I think it's ok to just ignore the `--print-location` argument in the case of `listing` and only apply it to `get`